utility/utils.js

```diff
@@ -130,14 +130,13 @@ class Utils {
    * @inner */
   static async ensureAuthenticated(req, res, next) {
     if (
-      req.body[0] != null &&
-      req.body[0].user_api != null &&
-      req.body[0].user_id
+      req.get("user-id") &&
```

Comment:
I was thinking to ensure backwards compatibility with certain apps (such as discord bots), do you think it would be possible to include an or for this instead?
```
if (
      req.body[0] != null &&
      ((req.body[0].user_api != null &&
      req.body[0].user_id) ||
      (req.get("user-id") &&
      req.get("user-api")))
```

I believe this would also require changes in G5V, which I believe should at least be given a chance to get some changes as well :)

Reply:
Yeah I already thought about that and you are right: This would be a breaking change but I was curious how you would like to handle it. I will implement the backwards compatability ^^

Reply:
Done in 290cfb9